azure-pipelines.yml

```diff
@@ -15,6 +15,16 @@ jobs:
         version: '1.17'
     - checkout: self
       fetchDepth: 1
+    - task: Bash@3
+      displayName: Install Dependencies
+      inputs:
+        targetType: inline
+        script: |
+          # install qemu
+          choco install qemu --version=2020.06.12
+          # install binaryen
+          wget https://github.com/WebAssembly/binaryen/releases/download/version_101/binaryen-version_101-x86_64-windows.tar.gz
```

Comment:
Apparently wget isn't included on this Windows version: https://github.com/actions/virtual-environments/blob/main/images/win/Windows2016-Readme.md
You may have better luck with curl, or by installing emscripten using Scoop instead: ScoopInstaller/Main#2740 (installing Scoop should be pretty fast).